heron/api/tests/scala/com/twitter/heron/streamlet/scala/impl/StreamletImplTest.scala

```diff
@@ -50,7 +56,7 @@ class StreamletImplTest extends BaseFunSuite {
       .setNumPartitions(20)
 
     supplierStreamlet
-      .map[Double]((num: Double) => num * 10)
+      .map[String]((num: Double) => (num * 10).toString)
```

Comment:
Ideally we should support this syntax.

.map[String] { num: Double =>
  (num * 10).toString
}

Not a blocker for now. We can add this support in the next version.